stamina-json/src/test/scala/stamina/json/JsonTestDomain.scala

```diff
@@ -9,9 +12,26 @@ object JsonTestDomain {
   // ==========================================================================
 
   case class ItemV1(id: ItemId, name: String)
+
+  object ItemV1 {
```

Comment:
Ah, so these are now required because we dropped the dependency on spray-json-shapeless (since that's not released for 2.13). Seems reasonable.